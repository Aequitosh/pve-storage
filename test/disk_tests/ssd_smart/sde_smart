smartctl 6.4 2014-10-07 r4002 [x86_64-linux-4.4.19-1-pve] (local build)
Copyright (C) 2002-14, Bruce Allen, Christian Franke, www.smartmontools.org

=== START OF READ SMART DATA SECTION ===
SMART overall-health self-assessment test result: PASSED

SMART Attributes Data Structure revision number: 10
Vendor Specific SMART Attributes with Thresholds:
ID# ATTRIBUTE_NAME          FLAGS    VALUE WORST THRESH FAIL RAW_VALUE
177 Wear_Leveling_Count     ------   000   000   000    -    0
230 Unknown_SSD_Attribute   PO--C-   100   100   000    -    100
231 Temperature_Celsius     ------   091   091   011    -    4294967296
233 Media_Wearout_Indicator -O--CK   000   000   000    -    43362
                            ||||||_ K auto-keep
                            |||||__ C event count
                            ||||___ R error rate
                            |||____ S speed/performance
                            ||_____ O updated online
                            |______ P prefailure warning
