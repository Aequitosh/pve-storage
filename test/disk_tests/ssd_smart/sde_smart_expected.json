{
    "attributes" : [
	{
	    "worst" : 0,
	    "threshold" : 0,
	    "name" : "Wear_Leveling_Count",
	    "value" : 0,
	    "id" : "177",
	    "raw" : "0",
	    "flags" : "------",
	    "fail" : "-"
	},
	{
	    "worst" : 100,
	    "name" : "Unknown_SSD_Attribute",
	    "threshold" : 0,
	    "id" : "230",
	    "fail" : "-",
	    "flags" : "PO--C-",
	    "raw" : "100",
	    "value" : 100
	},
	{
	    "worst" : 91,
	    "threshold" : 11,
	    "name" : "Temperature_Celsius",
	    "id" : "231",
	    "flags" : "------",
	    "raw" : "4294967296",
	    "fail" : "-",
	    "value" : 91
	},
	{
	    "worst" : 0,
	    "threshold" : 0,
	    "name" : "Media_Wearout_Indicator",
	    "id" : "233",
	    "flags" : "-O--CK",
	    "raw" : "43362",
	    "fail" : "-",
	    "value" : 0
	}
    ],
    "type" : "ata",
    "health" : "PASSED"
}
